include/ublksrv_priv.h

```diff
@@ -81,6 +81,11 @@ static inline void ublksrv_set_sqe_cmd_op(struct io_uring_sqe *sqe, __u32 cmd_op
 	addr[1] = 0;
 }
 
+static inline void ublksrv_log(int priority, const char *fmt, ...)
+	__attribute__ ((format (printf, 2, 3)));
+static inline void ublksrv_printf(FILE *stream, const char *fmt, ...)
+	__attribute__ ((format (printf, 2, 3)));
+
 #ifdef DEBUG
```

Comment:
Just wondering why you don't change the two inside the #ifdef DEBUG block? So we can avoid the extra two declarations.

Reply:
OK, then looks fine, then the patch is merged by adding the above description to commit log.

Thanks,

Reply:
Thank you! If I make future contributions, I'll do my best to include more details into the commit log.